float32 targetKph
bool estopEng
